angular/projects/catalyst/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "peerDependencies": {
     "@angular/core": ">=12.0.0",
-    "rxjs": ">=6.6.0"
+    "rxjs": ">=6.6.0",
+    "@haiilo/catalyst": ">=0.0.1"
```

Comment:
Thinking about it, I guess it is not a peerDependency, as the core and angular wrapper a closely coupled. So it would rather be a real dependency with a fixed version. Upside: No need to specify the dependency on COYO side anymore.

Reply:
moved it to dependencies.